docs/modules/develop/pages/classes/commands.adoc

```diff
@@ -0,0 +1,84 @@
+= Commands
+
+A command is custom class that manages the logic between your input, usually a xref:develop:classes/forms.adoc[form] object, a object and an actor.
+
+Usually, a command is used to create, update or delete a object.
+
+Commands are located in the `app/commands/decidim/<my_plugin>` directory, and named: `<my_command>.rb`.
+
+An example of a command class would:
+
+```ruby
+# frozen_string_literal: true
+
+module Decidim
+  module MyPlugin
+    # A command with the business logic to invite an user to an organization.
+    class CreateMyResource < Decidim::Command
+      # Public: Initializes the command.
+      #
+      # form
+      # resource
+      def initialize(form, resource)
+        @form = form
+        @resource = resource
+      end
+
+    def call
+      return broadcast(:invalid) if form.invalid?
+
+      transation do
+         create_resource
+      end
+      broadcast(:ok)
+    end
+
+    private
+
+    attr_reader :form, :resource
+
+    def create_resource
+      Decidim.traceability.create!(
+        resource,
+        form.current_user,
+        **attributes,
+        visibility: "public-only"
+      )
+    end
+
+    # this is mapping of
+    # ActiveRecord::attribute => form.attribute
+    def attributes
+      {
+        title: form.title,
+        description: form.description,
+        resource: form.resource
+      }
+    end
+  end
+end
+
+```
+
+
+Sometimes, you may need to extend a `Decidim` supplied command, then you can either override the `attributes` method, either extend it with a `super` call.
```

Comment:
I think all the overrides should be in the same page, probably https://docs.decidim.org/en/v0.27/customize/code, and then reference from the classes documentation to that page. The other way around would be OK to me, as long as it's consistent between the different strategies and classes.  

```suggestion
Sometimes you may need to extend a `Decidim` supplied command, then you can either override the `attributes` method, either extend it with a `super` call.
```